<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-LBN8TGD" ignored="0" name="RetryMulitple" tests="10" failures="1" timestamp="29 Mar 2019 11:42:27 GMT" time="0.13" errors="0">
  <testcase name="run" time="0.012" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.002" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.009" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.004" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.003" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.002" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.002" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.003" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.002" classname="testng.TestRetry1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.003" classname="testng.TestRetry1">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng.TestRetry1.run(TestRetry1.java:11)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- run -->
</testsuite> <!-- RetryMulitple -->
